pom.xml

```diff
@@ -109,6 +110,13 @@
           <version>${grpc.version}</version>
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-tcnative-boringssl-static</artifactId>
+          <version>${netty-tcnative.version}</version>
+          <classifier>${os.detected.classifier}</classifier>
+          <scope>test</scope>
```

Comment:
no netty-tcnatyve is not required to run jetcd, it is an optional dependency if you want to use a fast ssl implementation based on openssl.